src/Core/hcoio_read_std_mod.F90

```diff
@@ -1131,35 +1134,41 @@ SUBROUTINE HCOIO_Read( HcoState, Lct, RC )
        ENDIF
 
        ! Prompt a warning if thisUnit is not recognized as unitless.
-       IF ( Flag /= 0 ) THEN
+       ! Only do this if verbose since unitless is used to force bypass of unit conversion,
+       ! such as for reading non-emissions data, .e.g. restart file concentrations.
+       ! Print this message to HEMCO log (if using) rather than standard out to avoid
+       ! cluttering the regular log file.
+       IF ( Flag /= 0 .AND. HcoState%Config%doVerbose ) THEN
           MSG = 'Data is treated as unitless, but file attribute suggests ' // &
-                'it is not: ' // TRIM(thisUnit) // '. File: ' // TRIM(srcFile)
-          CALL HCO_WARNING( HcoState%Config%Err, MSG, RC )
+               'it is not: ' // TRIM(thisUnit) // '. File: ' // TRIM(srcFile)
+          IF ( HcoState%Config%doVerbose ) CALL HCO_WARNING( MSG, LUN=HcoState%Config%hcoLogLUN)
        ENDIF
 
        ! Verbose mode
-       IF ( HCO_IsVerb( HcoState%Config%Err ) ) THEN
+       IF ( HcoState%Config%doVerbose ) THEN
           WRITE(MSG,*) 'Based on srcUnit attribute (', TRIM(Lct%Dct%Dta%OrigUnit), &
                        '), no unit conversion is performed.'
-          CALL HCO_MSG(HcoState%Config%Err,MSG)
+          CALL HCO_MSG(MSG,LUN=HcoState%Config%hcoLogLUN)
        ENDIF
 
     ! Convert to HEMCO units in all other cases.
     ELSE
 
        ! For zero unit tolerance, make sure that thisUnit matches
        ! with unit set in configuration file. For higher unit
-       ! tolerances, prompt a level 3 warning.
+       ! tolerances, prompt a level 3 warning. Only print the warning
+       ! if verbose since most instances of unit mismatches are normal,
```

Comment:
@lizziel: Change `level 3 warning` to `warning`.  The level 3 was a holdover from the numeric verbose values.

Reply:
Now fixed.